languages/php/src/BitwardenLib.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Bitwarden\Sdk;
+
+use Bitwarden\Sdk\Schemas\ClientSettings;
+use Bitwarden\Sdk\Schemas\Command;
+use FFI;
+use Swaggest\JsonDiff\Exception;
+use Swaggest\JsonSchema\JsonSchema;
+
+
+class BitwardenLib
+{
+    public FFI $ffi;
+    public FFI\CData $handle;
+
+    /**
+     * @throws \Exception
+     */
+    public function __construct()
+    {
+        $lib_file = null;
+
+        if (PHP_OS === 'WINNT') {
+            $lib_file = '/lib/windows-x64/bitwarden_c.dll';
+            if (file_exists($lib_file) == false) {
+                $lib_file = __DIR__.'../../../../target/debug/bitwarden_c.dll';
+            }
+        } elseif (PHP_OS === 'Linux') {
+            $lib_file = '/lib/ubuntu-x64/libbitwarden_c.so';
+            if (file_exists($lib_file) == false) {
+                $lib_file = __DIR__.'../../../../target/debug/libbitwarden_c.so';
+            }
+        } elseif (PHP_OS === 'Darwin') {
+            $architecture = trim(exec('uname -m'));
+            if ($architecture === 'x86_64' || $architecture === 'amd64') {
+                $lib_file = __DIR__.'/lib/macos-x64/libbitwarden_c.dylib';
+            } elseif ($architecture === 'arm64') {
+                $lib_file = __DIR__.'/lib/macos-arm64/libbitwarden_c.dylib';
+            }
+            if (file_exists($lib_file) == false) {
+                $lib_file = __DIR__.'../../../../target/debug/libbitwarden_c.dylib';
```

Comment:
I think these directories are going one directory too high and should instead be:
```suggestion
                $lib_file = __DIR__.'/../../../target/debug/libbitwarden_c.dylib';
```

Otherwise we're looking for the file in a directory outside the sdk repo.

The same applies to the other operating systems.

Reply:
thank you, this was a mistake on my part.